sdks/python/apache_beam/typehints/schemas_test.py

```diff
@@ -511,10 +580,8 @@ def assert_namedtuple_equivalent(self, actual, expected):
     #for attr in dir(expected):
     #  self.assertEqual(getattr(actual, attr), getattr(expected, attr))
 
-  @parameterized.expand([
-      (fieldtype_proto, )
-      for fieldtype_proto in get_test_beam_fieldtype_protos()
-  ])
+  @parameterized.expand([(fieldtype_proto, ) for fieldtype_proto in
+                         get_test_beam_fieldtype_protos_with_logical_types()])
```

Comment:
This replaces the `get_test_beam_fieldtype_protos`. Can you add both `get_test_beam_fieldtype_protos_with_logical_types` and `get_test_beam_fieldtype_protos` to the tests?

Reply:
I defined it cumulatively:
```
def get_test_beam_fieldtype_protos_with_logical_types():
  logical_types = ...
  selected_schemas_with_logical_types = ...
  return get_test_beam_fieldtype_protos(
  ) + selected_schemas_with_logical_types + logical_types
```
But I can make that change for clarity?